tuxedo-template-runtime/src/lib.rs

```diff
@@ -295,26 +294,26 @@ impl From<runtime_upgrade::VerifierError> for OuterVerifierError {
 impl Verifier for OuterVerifier {
     type Error = OuterVerifierError;
 
-    fn verify(
+    fn verify<R: Redeemer>(
         &self,
-        input_data: &[DynamicallyTypedData],
-        output_data: &[DynamicallyTypedData],
+        inputs: &[Output<R>],
+        outputs: &[Output<R>],
     ) -> Result<TransactionPriority, OuterVerifierError> {
         Ok(match self {
-            Self::Money(money) => money.verify(input_data, output_data)?,
-            Self::FreeKittyVerifier(free_breed) => free_breed.verify(input_data, output_data)?,
+            Self::Money(money) => Verifier::verify(money, inputs, outputs)?,
```

Comment:
I'm surprised this change is necessary. Of course we are using the new, more general, `Verifier` trait, but I didn't expect that we should have to explicitly mention it as long as the `SimpleVerifier` is not in scope.

Reply:
You are correct I had initially the `SimpleVerifier` in scope. Am changing